### DISCUSSION TITLE
Using Boxes inside Custom Plugin: no display

### DISCUSSION THREAD

**Discussion by gionatan** [OFFLINE-GmbH/oc-boxes-support#103] [Help] [answered]
![Schermata del 2024-02-06 18-30-28](https://github.com/OFFLINE-GmbH/oc-boxes-support/assets/7984577/47ef5485-4509-4843-92aa-42be05821463)

Hi, we are trying to integrate Boxes inside a custom plugin model following the docs but we have this glitch in rendering. No errors in console. Thank you for any suggestion about it.

**Comment by tobias-kuendig** [comment]
Try putting the editor on a separate tab and also give it `span: adaptive`

**Comment by tobias-kuendig** [comment, marked as answer]
I have updated the docs to include the tab reference:
https://docs.boxes.offline.ch/use-cases/usage-in-plugins.html

The way October's forms are built it is easiest to place the editor on a separate tab with span: adaptive.

Thank you for reporting this, @gionatan .

**Reply by gionatan** [reply]
Ok, solved when I put it inside a tab, thank you!
